### PR TITLE
468a831d 56aa 4942 9c80 fb8405a738f6

### PR DESCRIPTION
**Pull request by GrikTad**
I add ImageClassifier.py and Dataset.py files. On ImageClassifier file is declared our model and on Dataset.py file is declared  our dataset.

### CODE REVIEW

File: ImageClassifier.py
Comment (Barsegh-A) [inline ImageClassifier.py:1]: Since we have already TrainEval class that contains the scripts for training, there is no need to include backpropagation and optimize methods in nn.Module classes. Also though the network might be an appropriate from technical perspective, but I think it it too small to get satisfactory results.

File: data/Dataset.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import torch
+import PIL
+class LandUseDataset(torch.utils.data.Dataset):
```
Comment (Barsegh-A) [inline data/Dataset.py:3]: Since I am going to change how the processed data info is stored (currently in .csv, will change to json, with each image having its label a one-hot-vector already, so there is no need for further processing)
